grafanalib/core.py

```diff
@@ -853,7 +860,7 @@ def to_json_data(self):
 class Dashboard(object):
 
     title = attr.ib()
-    rows = attr.ib()
+    struct = attr.ib()
```

Comment:
This will break every existing user of Dashboard, right?
Is there a way to add what you want without breaking existing things?